content/W2015/W15-event-research-showcase.md

```diff
@@ -1,7 +1,7 @@
 Title: Research Showcase and Mixer
 Date: 2015-02-04 17:30
 Category: Events
-Tags: talks, academia, social
+Tags: talks, academic, social
```

Comment:
1. I checked every tags in every pages, and there is only 1 "academia" and multiple "academic"(s). So, I was assuming that we actually want to use academic. However, if you want it to be academia, then I don't mind changing the others.
2. I can go through everything again this week and add all the missing tags, but I'm not too sure what you mean by 2 commits.
Thank you for the comments!